com.microsoft.mrtk.accessibility/Utilities/ReaderView.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.MixedReality.Toolkit.Accessibility
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    [Flags]
+    public enum ReaderView : ushort
```

Comment:
ditto on `ushort`

Reply:
same reason as ObjectClassification, we do not need 32 bits of data here...

also, considering the intent of how this will be used, i will likely get rid of the [Flags] attribute and make this a standard enum

Reply:
removed the [Flags] attribute. still a ushort ;)

Reply:
See above comment regarding design guidelines surrounding non-DWORD-sized managed objects

Reply:
done